lib/diagnostic.js

```diff
@@ -38,21 +48,31 @@ let owner = null;
 // Attributes: name, species, breed, vocalization
 // Methods: vocalize
 
-// Replace null with your answer
-let pet = null;
-
+// Replace null with your answe
+let Pet = {
+  name = "blue Jay",
+  species = "bird",
+  breed = "Blue male jay bird....",
+  vocalization = "Raor!!!!",
+  }
```

Comment:
Looks good, but I think you forgot to add a method that returns your `vocalization`. Something like this would work: `vocalize: function () {return this.vocalization}`